File: CaseStudies/noPCM/src/Cpp/OutputFormat.cpp
#include "OutputFormat.hpp"

#define _USE_MATH_DEFINES
#include <algorithm>
#include <iostream>
#include <fstream>
#include <iterator>
#include <string>
#include <math.h>
#include <sstream>
#include <limits>
#include <vector>

using std::string;
using std::vector;
using std::ifstream;
using std::ofstream;

void write_output(vector<double> T_W) {
    ofstream outputfile;
    outputfile.open("output.txt", std::fstream::out);
    outputfile << "T_W = [";
    for (int list_i1 = 0; list_i1 < (int)(T_W.size()) - 1; list_i1++) {
        outputfile << T_W.at(list_i1);
        outputfile << ", ";
    }
    if ((int)(T_W.size()) > 0) {
        outputfile << T_W.at((int)(T_W.size()) - 1);
    }
    outputfile << "]" << std::endl;
    outputfile.close();
}

File: CaseStudies/noPCM/src/Julia/OutputFormat.jl
module OutputFormat

function write_output(E_W::Float64, T_W::Array{Float64})
    outputfile = open("output.txt", "w")
    print(outputfile, "T_W = ")
    println(outputfile, T_W)
    print(outputfile, "E_W = ")
    println(outputfile, E_W)
    close(outputfile)
end

end